src/olympia/blocklist/cron.py

```diff
@@ -0,0 +1,25 @@
+import json
+import tempfile
+
+import olympia.core.logger
+from olympia.lib.kinto import KintoServer
+
+from .mlbf import generate_mlbf, get_mlbf_key_format
+from .utils import KINTO_BUCKET, KINTO_COLLECTION_MLBF
+
+log = olympia.core.logger.getLogger('z.cron')
+
+
+def upload_mlbf_to_kinto():
+    server = KintoServer(KINTO_BUCKET, KINTO_COLLECTION_MLBF)
+    stats = {}
+    key_format = get_mlbf_key_format()
+    bloomfilter = generate_mlbf(stats, key_format)
+    with tempfile.NamedTemporaryFile(delete=False) as filter_file:
```

Comment:
Why keep it around ?

Reply:
https://github.com/mozilla/addons-server/pull/13659/commits/d34fd01b586cafd8a53855d9b5dc1c1e9056074a